src/lib/components/forms/form/form-components/slider/slider.component.html

```diff
@@ -0,0 +1,20 @@
+<label *ngIf="name" class="mt-2 mb-1" [style.color]="'var(--color-structure-8)'">{{ name }}</label>
```

Comment:
```suggestion
<label *ngIf="name" class="mt-2 mb-1" [style.color]="'var(--label-color)'">{{ name }}</label>
```
Aber gerne auch über ne eigene Klasse das styling handeln.
Zum Beispiel würde ich auch noch das bottom-margin auf 0 und den Text auch nur in capitals (so wie bei den material inputs) machen.

Reply:
ok